File: src/Omnia.ProcessManagement.Web/client/fx/models/data/processes/Link.ts
import { MultilingualString, GuidValue } from '@omnia/fx-models';
import { Enums } from '../../Enums';

export interface Link {
    id: GuidValue;
    title: MultilingualString;
    linkType: Enums.LinkType;
    url: string;
    openNewWindow: boolean;

    //client-side
    multilingualTitle?: string
}

File: src/Omnia.ProcessManagement.Models/Processes/Link.cs
using Omnia.Fx.Models.Language;
using Omnia.ProcessManagement.Models.Enums;
using System;
using System.Collections.Generic;
using System.Text;

namespace Omnia.ProcessManagement.Models.Processes
{
    public class Link
    {
        public Guid Id { get; set; }
        public MultilingualString Title { get; set; }
        public string Url { get; set; }
        public LinkType LinkType { get; set; }
        public bool OpenNewWindow { get; set; }
    }
}
